lib/beaker/host/mac/exec.rb

```diff
@@ -27,4 +27,11 @@ def ssh_permit_user_environment
     ssh_service_restart()
   end
 
+  # Checks if selinux is enabled
+  #
+  # @return [Boolean] true if selinux is enabled, false otherwise 
```

Comment:
@tvpartytonight @kevpl @pinkypie should i change the doc here to say this method will only return false? (and also in the exec.rb for the windows host?) would there be any value in creating a selinux_available? method, to be called before querying if selinux is enabled? or would that be superfluous?